__init__.py

```diff
@@ -3,9 +3,10 @@
 from trytond.pool import Pool
 from . import account
 from . import currency
-from . import tax
 from . import invoice
+from . import party
 from . import product
+from . import tax
```

Comment:
I guess you're moving "tax" at the end to keep alphabetical order, and although I don't see anything wrong with it but there's no requirement in PEP8 that imports are done in alphabetical order.

Neither or model registration which sometimes needs to happen in a given different order.

Reply:
I know, it was unconscious. We could revert this part. It's not the important.